protocols/v2/roles-logic-sv2/src/channel_logic/channel_factory.rs

```diff
@@ -746,23 +747,26 @@ impl ChannelFactory {
         if tracing::level_enabled!(tracing::Level::DEBUG)
             || tracing::level_enabled!(tracing::Level::TRACE)
         {
-            println!("Bitcoin target: {:?}", bitcoin_target);
+            debug!("Bitcoin target: {:?}", bitcoin_target);
             let upstream_target: binary_sv2::U256 = upstream_target.clone().try_into().unwrap();
             let mut upstream_target = upstream_target.to_vec();
             upstream_target.reverse();
             debug!("Upstream target: {:?}", upstream_target.to_vec().to_hex());
-            let mut hash = hash;
-            hash.reverse();
-            debug!("Hash: {:?}", hash.to_vec().to_hex());
+            let mut print_hash = hash.clone();
+            print_hash.reverse();
```

Comment:
Seems like we shouldn't reverse the hash before it's being used to check the `bitcoin_target` right?

Reply:
[u8;32] is copy so it reverse just a copy

Reply:
yep - i figured that out :) - clippy warned about it - i changed it back.